admin/src/app/home/orders/stock-orders/generated/generated.component.html

```diff
@@ -260,3 +269,110 @@
     <!--/.col-->
   </div>
 </div>
+
+<div bsModal #addProductModal="bs-modal" class="modal fade " tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
```

Comment:
Make this a separate component.. and the add this in fulfill and receive components also